bigchaindb/pipelines/block.py

```diff
@@ -73,10 +74,15 @@ def validate_tx(self, tx):
 
         # If transaction is not valid it should not be included
         try:
+            if tx.operation == Transaction.GENESIS:
+                blocks_count = backend.query.count_blocks(self.bigchain.connection)
```

Comment:
I would replace the trip to the database with a line or two of comments based on what Rodolphe just wrote, so it's clear to the reader why the trip isn't necessary.